### ISSUE TITLE
Breaking Changes in kong dependency

### ISSUE THREAD

**Issue by DomenicDev** [Felix-Alexander-Jacobsen/devenv#1]
Hi, 

Thanks a lot for your great devenv tool.  

A few days ago, a new version of the kong chart has been released (v2.16.0), which, however, comes with breaking changes. The changes made break the script that deploys kong, which serves as an api-gateway. You can reproduce the issue by simply running the script, but make sure kong is not running, so it will force a redeployment.

A temporary solution is to fallback to an older chart version. For example, I replaced line 56 and 105 with the following content in order to fallback to version v2.15.0.

```
create_helm_deployment "kong" "api-gateway" "kong/kong" "--set ingressController.installCRDs=false --set admin.enabled=true --set admin.http.enabled=true --set admin.type='ClusterIP' --version 2.15.0"
```

The following issue reported by a developer talks about the breaking changes: https://github.com/Kong/charts/issues/730

Thanks a lot for looking into it.

Best regards
Domenic

**Comment by Felix-Alexander-Jacobsen** [comment]
Hi Domenic!

Thank you for reporitng the issue! Also, I'm glad to hear that you like the tool :).

It seems that the Kong team has released fixes for their issue:
https://github.com/Kong/charts/commit/cf4be8ef35d505b009a6d7d15a7df055e12dd349
https://github.com/Kong/charts/commit/7f6462aebe6e7be7d3fa322e93ed0c83f160185e

The situation seems to be resolved in kong-2.16.2.

Nevertheless I have published a fix which should reduce friction from such changes in Kong: https://github.com/Felix-Alexander-Jacobsen/devenv/commit/2caebd1826772fb1689a484dac3ce6abcb83f813

Feel free pull the update!

**Comment by DomenicDev** [comment]
Hello again,

Thanks a lot for your quick response. I can confirm that the latest release from yesterday fixes the issue. 

Thanks again for your time and effort. Keep up the good work 👍 

Cheers, Domenic
